tests/ui_tools/test_popups.py

```diff
@@ -971,7 +1073,11 @@ def test_height_noreactions(self):
             )
         ],
     )
-    def test_height_reactions(self, message_fixture, to_vary_in_each_message):
+    def test_height_reactions(
+        self,
+        message_fixture: Message,
+        to_vary_in_each_message: Message,
```

Comment:
Is this the only way that the `.update` can work? I suppose this is OK for now.

Reply:
We can also use another `TypedDict` identical to `Message` but `Message` is already non-total so I haven't done that. Other than that, there's the option to inline the dict, but that's not possible here. I referred to this previous discussion on CZO: [#zulip-terminal > update keys of TypedDict](https://chat.zulip.org/#narrow/stream/206-zulip-terminal/topic/update.20keys.20of.20TypedDict).